sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -188,15 +188,15 @@ Recognize data from US sales receipts using a prebuilt model.
 ```C# Snippet:FormRecognizerSampleRecognizeReceiptFileStream
 using (FileStream stream = new FileStream(receiptPath, FileMode.Open))
 {
-    RecognizedReceiptCollection receipts = await client.StartRecognizeReceipts(stream).WaitForCompletionAsync();
+    RecognizedFormCollection receipts = await client.StartRecognizeReceipts(stream).WaitForCompletionAsync();
 
     // To see the list of the supported fields returned by service and its corresponding types, consult:
     // https://westus2.dev.cognitive.microsoft.com/docs/services/form-recognizer-api-v2-preview/operations/GetAnalyzeReceiptResult
```

Comment:
I just remembered that yesterday we agreed on making this an aka.ms link so later on we could update it. 
i.e. Python created https://aka.ms/azsdk/python/formrecognizer/receiptfields

Reply:
For Preview 4: #12968